pages/common/lzop.md

```diff
@@ -0,0 +1,16 @@
+# lzop
+
+> Compress or decompress files with LZO compression.
+> More information: <https://www.lzop.org/>.
+
+- Compress a file into a new file with the .lzo suffix:
+
+`lzop {{file}}`
+
+- Decompress a file:
+
+`lzop -d {{file}}.lzo`
+
+- Compress a file, while specifing the compression level. 0 = Worst, 9 = Best (Default level is 3):
```

Comment:
```suggestion
- Compress a file, while specifing the compression level (0 being worst, 9 best, and 3 default):
```

Reply:
FYI: I copied this line from pages/zstd.md. If the line gets changed here we should maybe also change zstd's tldr-page.

Reply:
Then we should change it in the both files. If you want, you can do it here as well.